include/deal.II/matrix_free/fe_point_evaluation.h

```diff
@@ -1273,7 +956,7 @@ class FEPointEvaluation
   AlignedVector<vectorized_value_type> solution_renumbered_vectorized;
 
   /**
-   * Temporary array for the use_face_path path (scalar).
+   * Temporary array for the face path (scalar).
```

Comment:
Shouldn't this be part of the derived class?

Reply:
Currently, we initialize this field during `setup` which is called in the constructor of `FEPointEvaluationBase` but we could and should shift this responsibility to `FEFacePointEvaluation`.